apps/web/src/views/Home/components/Hero.tsx

```diff
@@ -82,29 +88,90 @@ const Hero = () => {
         id="homepage-hero"
       >
         <Flex flex="1" flexDirection="column">
-          <Heading as="h1" scale="xxl" color="secondary" mb="24px">
+          <Heading as="h1" scale="xxl" color="secondary" mb="12px">
             {t('IceCream brings people together')}
           </Heading>
+
+          <div style={{ maxWidth: 225, height: 75 }}>
+            <div className="certik-emblem" data-id="91b5fa0e">
+              <a href="https://skynet.certik.com/projects/icecreamswap?utm_source=SkyEmblem&utm_campaign=icecreamswap&utm_medium=link">
+                View project at certik.com
+              </a>
+            </div>
+          </div>
+
           <Heading as="p" scale="md" mb="24px">
             {/* todo: make links display somewhat like the text, maybe a small underline or so. */}
-            <Link href="https://icecreamswap.com/swap" display="inline-flex">DEX</Link>{", "}
-            <Link href="https://icecreamswap.com/swap" display="inline-flex">{t('DEX Aggregator')}</Link>{", "}
-            <Link href="https://chainlist.org" display="inline-flex">RPC</Link>{", "}
-            {t('Block Explorer')}{", "}
-            <Link href="https://icecreamswap.com/kyc" display="inline-flex">KYC</Link>{", "}
-            <Link href="https://icecreamswap.com/launchpad" display="inline-flex">{t('Launchpad')}</Link>{t(' and ')}
-            <Link href="https://icecreamswap.com/bridge" display="inline-flex">{t('Bridge')}</Link>{t(' on ')}
-            <Link href="https://base.org/" display="inline-flex">Base</Link>{", "}
-            <Link href="https://coredao.org/" display="inline-flex">Core DAO</Link>{", "}
-            <Link href="https://www.telos.net/" display="inline-flex">Telos</Link>{", "}
-            <Link href="https://bitgert.com/" display="inline-flex">Bitgert</Link>{", "}
-            <Link href="https://xinfin.org/" display="inline-flex">XDC</Link>{", "}
-            <Link href="https://shardeum.org/" display="inline-flex">Shardeum</Link>{", "}
-            <Link href="https://shimmer.network/" display="inline-flex">Shimmer</Link>{", "}
-            <Link href="https://www.bnbchain.org/en/smartChain" display="inline-flex">BSC</Link>{", "}
-            <Link href="https://www.xo-dex.com/" display="inline-flex">Xodex</Link>{", "}
-            <Link href="https://dogechain.dog/" display="inline-flex">Dogechain</Link>{t(' and ')}
-            <Link href="https://www.fuse.io/" display="inline-flex">Fuse</Link>.
+            <Link href="https://icecreamswap.com/swap" display="inline-flex">
```

Comment:
all these re-stylings are not required. Please make the pull requests minimal with only required modifications, no new formattings etc.

Reply:
looks like the CertiK emblem does not show at all, please test this and also make sure it works on multiple browsers and mobile.

Reply:
it's not new formatting, it's the settings in prettier. please check the Certix, it works fine on my end.